src/index.html

```diff
@@ -11,6 +11,19 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form form--big" action="/"
+    method="post" data-qa="big">
```

Comment:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/54860641/7c38129d-e368-43b6-902e-82ac0d2f1df5)
Fix this in all code.